config/config.default.yaml

```diff
@@ -84,9 +84,9 @@ co2_budget:
 
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#electricity
 electricity:
-  voltages: [200., 220., 300., 380., 500., 750.]
+  voltages: [220., 300., 330., 380., 400., 500., 750.]
```

Comment:
Can you provide some background why 200 kV was removed?

Reply:
Yes, I double-checked the voltage levels being used in Europe. Both JAO Static Grid Model and the ENTSO-E map show that that 200 kV is not a standardised value in operation. I then checked the few lines in OSM that were labeled as 200 kV lines, of which some were actually 20 kV lines (with a typo, trailing zero). Now the underlying OSM input data (at least for Europe) does not contain 200 kV lines anymore.

Reply:
Thanks for the explanation!